packages/service-clients/odsp-client/src/interfaces.ts

```diff
@@ -77,6 +77,18 @@ export interface OdspContainerAttachProps {
 	 * The file name of the Fluid file. If undefined, the file is named with a GUID.
 	 */
 	fileName: string | undefined;
+
+	/**
+	 * The ID of the item (file) to which the container is being attached.
+	 * When combined with eTag, this will trigger a conversion of an existing file to a Fluid file.
```

Comment:
How coupled are `itemId` and `eTag`? Would it make sense to combine them into a single object property? Or are they really independent?

Reply:
Are there public docs we can point to from the interface-level comment? Might be nice to make that relationship explicit in the docs.

Reply:
TBH I don't know of any docs, public or private, about this :/ I can add some doc comments to this though with my understanding